x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/WatcherService.java

```diff
@@ -139,8 +139,7 @@ public boolean validate(ClusterState state) {
         // template check makes only sense for non existing indices, we could refine this
         boolean hasValidWatcherTemplates = WatcherIndexTemplateRegistry.validate(state);
```

Comment:
I think we can remove this check entirely from here since it also checked on reload.

Reply:
Is it not worth doing the check for the debug log? I thought it might be useful so I left it. But I've never done any troubleshooting on this.

Reply:
possibly..but this happens on the cluster state listener thread and less work done there the better.